src/vcf/validator.cpp

```diff
@@ -20,6 +20,12 @@ namespace ebi
 {
   namespace vcf
   {
+    bool validate(const std::vector<char> &firstLine,
```

Comment:
Fair enough, I can be fine with the forward declaration as those will never be part of the public API.